src/main/scala/beam/router/Modes.scala

```diff
@@ -101,6 +103,13 @@ object Modes {
           TransportMode.transit_walk
         )
 
+    case object EMERGENCY_TRANSIT
+        extends BeamMode(
+          value = "emergency_transit",
+          Some(Right(TransitModes.TRANSIT)),
+          TransportMode.other
+        )
+
```

Comment:
Do we really need this `EMERGENCY_TRANSIT` mode? Why we cannot use just EMERGENCY even if there is a TRANSIT leg in it?